stacks/jupyterhub-pyspark-hdfs/jupyterlab.yaml

```diff
@@ -0,0 +1,67 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: jupyterlab
+  labels:
+    app: jupyterlab
+    stackable.tech/vendor: Stackable
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: jupyterlab
+      stackable.tech/vendor: Stackable
+  template:
+    metadata:
+      labels:
+        app: jupyterlab
+        stackable.tech/vendor: Stackable
+    spec:
+      serviceAccountName: default
+      containers:
+        - name: jupyterlab
+          image: oci.stackable.tech/sdp/spark-connect-client:3.5.5-stackable0.0.0-dev
+          imagePullPolicy: IfNotPresent
+          command:
+            - bash
+          args:
+            - -c
+            - /stackable/.local/bin/jupyter lab --ServerApp.token='{{ jupyterLabToken }}' --ServerApp.port=8080 --no-browser --notebook-dir /notebook
+          env:
+            - name: JUPYTER_PORT
+              value: "8080"
+          ports:
+            - name: http
+              containerPort: 8080
+          volumeMounts:
+            - mountPath: /notebook
+              name: notebook
+      initContainers:
+        - name: download-notebook
+          image: oci.stackable.tech/sdp/spark-connect-client:3.5.5-stackable0.0.0-dev
```

Comment:
This might change depending on the outcome mentioned in the dependent PR:

https://github.com/stackabletech/docker-images/pull/1071#issuecomment-2847048547

Reply:
yes, let's wait with merging this one

Reply:
```suggestion
          image: oci.stackable.tech/stackable/spark-connect-client:3.5.5-stackable0.0.0-dev
```

Reply:
Done in 3eeefe70d46c165d5c49e3918e325ceb2587ca8c